MEMBER_TRAVEL_FUND.md

```diff
@@ -113,8 +113,9 @@ Once the request has been approved, you must:
 
 1. Provide receipts as attachments in an email stating your name, the participation covered,
 and the total approved for reimbursement.
-2. Fill out and attach the [Expense Report](./expense-report-template.xls?raw=true) and rename as ExpenseReport-Node-YOURNAME-YYYYMM.
-3. Email this reimbursement request to travelapprovals@nodejs.org.
+2. Fill out and attach the [Expense Report](./expense-report-template.xls?raw=true) and rename as ExpenseReport-Node-EVENTNAME-YOURNAME-YYYYMM.
+3. Sign and date the file in PDF form, The sign line can be found in the lower left hand corner of the above document. The individual tasked with approve the expense will countersign before sending for processing. You may use an e-signature such as docusign or [the 'markup' feature in Preview](https://support.apple.com/guide/preview/fill-out-and-sign-pdf-forms-prvw35725/mac).
```

Comment:
> Sign and date the file in PDF form,

Huh??? Does that mean "Sign and date the form and return it as a PDF."?

(Also, should use a period, not a comma.)

Reply:
> The individual tasked with approve the expense will countersign before sending for processing.

Grammar error: approve -> approving

I wonder if the sentence can/should be removed? This is internal process stuff that the person submitting for reimbursement need not know.

Reply:
```suggestion
3. Sign the form, date the form, and return it as a PDF. The sign line can be found in the lower left hand corner of the above document. The individual tasked with approve the expense will countersign before sending for processing. You may use an e-signature such as docusign or [the 'markup' feature in Preview](https://support.apple.com/guide/preview/fill-out-and-sign-pdf-forms-prvw35725/mac).
```

Reply:
@namdeirf can you please address the comments above?

Reply:
```suggestion
3. Sign the form, date the form, and return it as a PDF. The sign line can be found in the lower left hand corner of the above document. The individual tasked with approving the expense will countersign before sending for processing. You may use an e-signature such as docusign or [the 'markup' feature in Preview](https://support.apple.com/guide/preview/fill-out-and-sign-pdf-forms-prvw35725/mac).
```

Reply:
I think the suggested comments need to be accepted before it can be merged.

Reply:
@Trott Is this alright? :)

Reply:
Seems OK to me.

Reply:
@namdeirf I still don't see this suggestion from @SofiEstevez having been accepted into the PR?